packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+/**
+ * Breadcrumb element represents  navigation
+ */
+
+@Component({
+  tag: 'datacom-breadcrumb',
+  styleUrl: 'datacom-breadcrumb.css',
+  shadow: true,
+})
+export class DatacomBreadcrumb {
+  @Prop() url?: string;
+  @Prop() text: string;
+
+  NavIcon = getSvg('drill-down', { class: 'dc-breadcrumb-icon' });
+
+  render() {
+    return (
+      <Host>
+        {!!this.url ? (
+          <div>
+            <a href={this.url} class="dc-breadcrumb">
+              {this.text}
+              <slot></slot> {this.NavIcon}
+            </a>
+            <slot></slot>
```

Comment:
Don't think you need both a <slot> and text prop. (They seem to be achieving the same thing.) You can probably scrap the text prop and leave the <slot> element in?

(If this was text prop was added to make it configurable with Storybook, have a look at "datacom-content-tag.stories.tsx". This story is an example of making the <slot> content configurable.)